chronos/transports/stream.nim

```diff
@@ -1049,7 +1051,8 @@ when defined(windows):
                 # was already scheduled, so we failing it not with OS error.
                 retFuture.fail(getServerUseClosedError())
               else:
-                retFuture.fail(getTransportOsError(err))
+                let errorMsg = osErrorMsg(err)
+                retFuture.fail(getConnectionAbortedError(errorMsg))
```

Comment:
Hmm, not sure if _all_ of the error codes should become an `ConnectionAbortedError`. Things like `WSANOTINITIALISED` would still be a usage error / a `TransportOsError`.